src/AdminBundle/EventSubscriber/EventAccessSubscriber.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of Eventbase API.
+ *
+ * (c) 2017–2018 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace AdminBundle\EventSubscriber;
+
+use AppBundle\Entity\Event;
+use EasyCorp\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Security\Core\Security;
+
+/**
+ * Class EventAccessSubscriber
+ */
+class EventAccessSubscriber implements EventSubscriberInterface
+{
+    /** @var Security */
+    private $security;
+
+    /**
+     * EventAccessSubscriber constructor.
+     *
+     * @param Security $security
+     */
+    public function __construct(Security $security)
+    {
+        $this->security = $security;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            EasyAdminEvents::PRE_PERSIST => ['setEventAccessFromUSer'],
```

Comment:
`setEventAccessFromUSer` should be `setEventAccessFromUser` (lowercase `s` in `User`).